.github/workflows/merge-python-requirements-upgrade-prs.yml

```diff
@@ -0,0 +1,101 @@
+name: Merge Python Requirements Upgrade PRs
+
+on:
+  schedule:
+    - cron: "0 7 * * *"
+
+jobs:
+  get_list_of_prs:
+    name: Get list of PRs
+    runs-on: ubuntu-20.04
+    outputs:
+      urls: ${{ steps.combined_urls.outputs.value }}
+    steps:
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+
+      - name: Checkout tools repo
+        uses: actions/checkout@v3
+        with:
+          repository: openedx/tubular
+          path: tubular
+
+      - name: Setup tubular
+        run: |
+          cd tubular
+          pip install -e .
+
+      - name: Get urls of all eligible prs
+        env:
+          GIT_TOKEN: ${{ secrets.requirements_bot_github_token }}
+        id: generate-matrix
+        run: |
+          echo "urls_edx=$(get_ready_to_merge_prs.py --org edx | sed '1d;2d;')" >> $GITHUB_OUTPUT
+          echo "urls_openedx=$(get_ready_to_merge_prs.py --org openedx | sed '1d;2d;')" >> $GITHUB_OUTPUT
```

Comment:
I thought get_ready_to_merge_prs.py was written for this action.  Why do we need to delete the first two lines of output?  I think it would be better to keep this script in this repo, and to make it more specific to its purpose.

Reply:
Yeah, moved here. Will create a PR in tubular to remove from there.